pluto/src/vis/button/aether/button.ts

```diff
@@ -28,11 +29,13 @@ export class Button extends aether.Leaf<typeof buttonStateZ, InternalState> {
 
   async afterUpdate(): Promise<void> {
     const { sink: sinkProps } = this.state;
+    if (this.internal.prevTrigger == null)
+      this.internal.prevTrigger = this.state.trigger;
     this.internal.sink = await telem.useSink(this.ctx, sinkProps, this.internal.sink);
 
-    if (this.state.trigger > this.prevState.trigger) {
-      this.internal.sink.set(true).catch(console.error);
-    }
+    if (this.state.trigger <= this.internal.prevTrigger) return;
+    this.internal.prevTrigger = this.state.trigger;
```

Comment:
Won't you want to set `prevTrigger` to `this.state.trigger` always? This will get blocked because of the return statement the line above.

Reply:
Yeah good point